mlflow/server/handlers.py

```diff
@@ -44,6 +43,10 @@ def _get_store():
     return _store
 
 
+def _get_request_json(flask_request=request):
```

Comment:
Pulled into its own method to simplify mocking `flask_request.get_json` output in testing